src/interpolate.jl

```diff
@@ -11,7 +11,7 @@ end
 
 # Interpolate data from input format to desired output format (CurvedMesh or UnstructuredQuadMesh version)
 function interpolate_data(::Val{:vtu}, input_data,
-                          mesh::Union{Trixi.CurvedMesh, Trixi.UnstructuredQuadMesh},
+                          mesh::Union{Trixi.CurvedMesh, Trixi.UnstructuredQuadMesh, Trixi.P4estMesh},
```

Comment:
If you need these mesh types in multiple places in Trixi2Vtk, you could change
```julia
import Trixi
```
to
```julia
using Trixi: CurvedMesh, UnstructuredQuadMesh, P4estMesh, TreeMesh
```
in `Project.toml`.

Reply:
Good point!